timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosRemoteClients.java

```diff
@@ -160,7 +161,11 @@
                                 .remotingClientConfig(() -> RemotingClientConfigs.DEFAULT)
                                 .shouldSupportBlockingOperations(false)
                                 .build()))
-                .map(proxy -> AtlasDbMetrics.instrumentWithTaggedMetrics(metrics().getTaggedRegistry(), clazz, proxy));
+                .map((host, proxy) -> AtlasDbMetrics.instrumentWithTaggedMetrics(
+                        metrics().getTaggedRegistry(),
+                        clazz,
+                        proxy,
+                        any -> ImmutableMap.of(AtlasDbMetricNames.TAG_REMOTE_HOST, host)));
```

Comment:
Is this risky in terms of metrics cardinality? I'm in the process of checking with the internal team that's responsible for this

Reply:
Checked internally: no, this is fine.